drivers/objectstore/aws/aws.go

```diff
@@ -0,0 +1,22 @@
+package aws
+
+import (
+	torpedoobjectstore "github.com/portworx/torpedo/drivers/objectstore"
+)
+
+const (
+	// DriverName is the name of the aws objectstore driver implementation
+	DriverName = "aws"
+)
+
+type aws struct {
```

Comment:
as discussed offline, we don't need separate drivers for each of provider

Reply:
Fixed and tested here:  https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/741/